open-company/open-company-web
# Manage members tweaks

Card: https://trello.com/c/I6bMPDxg

To test:
- open user dropdown menu
- [x] do you see Manage Team? Good
- [x] does it has the people icon? Good
- click it
- [x] does it open the Manage Team panel? Good
- [x] make sure the name is *Manage Team* and not ~Manage Memebers~
- close the settings panel
- now look at the links in bottom left corner
- [x] do you see Support below Invite Team (no ~Invite people~)? Good
- [x] do you see the chat bubbles icon for Support? Good
- [x] do you see the users icon for Invite People? Good
